FiftyOne.Pipeline.CloudRequestEngine/FlowElements/CloudRequestEngine.cs

```diff
@@ -191,14 +188,32 @@ public class CloudRequestEngine :
         /// be populated after a call to the cloud service as part of 
         /// object initialization.
         /// </summary>
-        public override IEvidenceKeyFilter EvidenceKeyFilter => _evidenceKeyFilter;
+        public override IEvidenceKeyFilter EvidenceKeyFilter {
+            get
+            {
+                if (_evidenceKeyFilter == null)
+                {
+                    GetCloudEvidenceKeys();

```

Comment:
See comment for the GetCloudProperties method.

Reply:
Changes applied.